tests/suites/os/tests/connectivity/index.js

```diff
@@ -108,8 +108,17 @@ module.exports = {
 								this.context.get().link,
 							);
 
-						// Rebooting the DUT
-						await this.context.get().worker.rebootDut(this.context.get().link);
+						test.comment(`Waiting for redsocks service to be active...`);
+						await this.context.get().utils.waitUntil(async () => {
+							return (
+								(await this.context
+									.get()
+									.worker.executeCommandInHostOS(
+										`systemctl is-active redsocks.service`,
+										this.context.get().link,
+									)) === 'active'
+							);
+						}, false);
```

Comment:
while we're here, we may want to reduce the wait time. In a failed test just now it took 40 minutes to timeout:
```
}, false, 5, 30000);
```

Will retry this 5 times instead of 20

Reply:
Yeah, I'm working on reproducing this one locally and I already made a similar change.